packages/orama/tests/search.test.ts

```diff
@@ -6,6 +6,38 @@ t.test('search method', t => {
   t.test('with term', async t => {
     const [db, id1, id2, id3, id4] = await createSimpleDB()
 
+    //https://github.com/oramasearch/orama/issues/480
+    //following testcase pass only if issue 480 is fixed.
+    t.test('should correctly match with tolerance . even if prefix doesnt match.', async t => { 
+      t.plan(2)
+
+      const db = await create({
+        schema: {
+          name: 'string',
+        },
+        components: {
+          tokenizer: {
+            stemming: true,
+            stopWords: englishStopwords,
+          },
+        },
+      })
+
+      await insert(db, { name:"Chris "})
+      await insert(db, { name:"Craig"})
+      await insert(db, { name:"Chxy"}) //create h node in radix tree.
+      await insert(db, { name:"Crxy"}) //create r node in radix tree.
```

Comment:
Can you try by adding:

```js
await insert(db, { name: "Dhris" })
```

and see if it matches `"Chris"` when tolerance is set to `1`?

Reply:
`     await insert(db, { name: "Dhris" })
      const result1 = await search(db, { term: 'Chris', tolerance: 1 })
      const result2 = await search(db, { term: 'Cgris', tolerance: 1 })
      t.equal(result1.count, 1)
      t.equal(result2.count, 0)// should only match on tolerance 2.
`
It works as expected.

Reply:
Can you please update the PR with this test?

Reply:
> Can you please update the PR with this test?

updated PR.